bin/install_grass.sh

```diff
@@ -50,24 +48,24 @@ fi
 USER_HOME="/home/$USER_NAME"
 
 TMP_DIR=/tmp/build_grass
-mkdir "$TMP_DIR"
+mkdir -p "$TMP_DIR"
 
 apt-get --quiet update
-apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal-grass
+apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal-grass unzip
 
-cp /usr/share/applications/grass82.desktop "$USER_HOME/Desktop/"
-chown -R $USER_NAME.$USER_NAME "$USER_HOME/Desktop/grass82.desktop"
+cp /usr/share/applications/grass8?.desktop "$USER_HOME/Desktop/Desktop GIS/"
+chown -R $USER_NAME:$USER_NAME "$USER_HOME/Desktop/Desktop GIS/grass8?.desktop"
```

Comment:
Please revert to copy to "$USER_HOME/Desktop/"
The Desktop setup is handled in https://github.com/OSGeo/OSGeoLive/blob/master/bin/install_icons_and_menus.sh

Reply:
Sure, done in b55d4d69efacb039f9b6ef3f8ae36a0e579387c6